src/assets/scss/custom.scss

```diff
@@ -18,3 +18,31 @@ $carousel-control-width: 3%;
     z-index: $zIndexContent;
   }
 }
+
+div.version-ribbon {
+  background-color: #b11;
+  overflow: hidden;
+  white-space: nowrap;
+  position: absolute;
+  height: 30px;
+  width: 260px;
+  left: -50px;
+  top: 65px;
+  transform: rotate(-45deg);
+  box-shadow: 0 0 10px #888;
+  z-index: 999;
+
+  span {
+    display: inline-block;
+    color: #fee;
+    font-weight: bold;
+    width: 200px;
+    margin: 1px 30px;
+    padding: 0;
+    vertical-align: middle;
+    text-align: center;
+    text-shadow: 0 0 5px #444;
+    text-size-adjust: none;
+    cursor: pointer;
+  }
+}
```

Comment:
```suggestion
  z-index: 999;
}

div.version-ribbon span {
    display: inline-block;
    color: #fee;
    font-weight: bold;
    width: 200px;
    margin: 1px 30px;
    padding: 0;
    vertical-align: middle;
    text-align: center;
    text-shadow: 0 0 5px #444;
    text-size-adjust: none;
    cursor: pointer;
}
```

Looks cleaner to me, but it's optional. Feel free to resolve yourself.

Reply:
Since this is an SCSS file, I think we can make use of nesting to avoid repetition of the selector and have everything related to the ribbon inside one block. Later this can be moved to the CSS

And since nested CSS is now also supported by browsers natively, I think we will soon get used to it and perceive it as cleaner.

When we move to Angular, everything inside the top level selector will be moved from the global style file to a component CSS file - the selector will not even be needed any more.

So I would like to keep it that way.